### PR TITLE
Catty-463 Implement SetBackgroundAndWaitBrick

### PR DESCRIPTION
**Pull request by Bomml**
Similar to the SetBackgroundBrick, this Brick should change the background of the current SpriteObject. In contrast to the SetBackgroundBrick, this Brick should wait with the execution of subsequent Bricks (of the Script the SetBackgroundAndWaitBrick is in) until all Bricks within a WhenBackgroundChangesScript have been executed (if such a Brick exists).

Small test program:
https://share.catrob.at/app/project/2ec20395-5c1b-11eb-a92d-005056a36f47

- [x] Include the name of the Jira ticket in the PR’s title
- [x] Verify that the Jira ticket is in the status *Ready for Development*
- [x] Include a summary of the changes plus the relevant context
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [x] Verify that the changes generate no compiler or linter warnings
- [x] Perform a self-review of the changes
- [x] Verify to commit no other files than the intentionally changed ones
- [x] Include reasonable and readable tests verifying the added or changed behavior
- [x] Confirm that new and existing unit tests pass locally
- [x] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [x] Stick to the project’s git workflow (rebase and squash your commits)
- [x] Verify that your changes do not have any conflicts with the base branch
- [x] After the PR, verify that all CI checks have passed
- [x] Post a message in the *#catty* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### CODE REVIEW

File: src/Catty/Views/Custom/CollectionViewCells/BrickCells/Look/SetBackgroundAndWaitBrickCell.swift [hunk 1]
```diff
@@ -0,0 +1,53 @@
+/**
+ *  Copyright (C) 2010-2020 The Catrobat Team
```
Comment (zwerg4) [inline src/Catty/Views/Custom/CollectionViewCells/BrickCells/Look/SetBackgroundAndWaitBrickCell.swift:2]: Please change to  *  Copyright (C) 2010-2021 The Catrobat Team

**Review by zwerg4 — CHANGES_REQUESTED** [review]
Looks good. please change the year that is wrong in some headers.

File: src/Catty/Views/Custom/CollectionViewCells/BrickCells/Look/SetBackgroundAndWaitBrickCell.swift (hunk 1 above)
Comment (Bomml) [reply src/Catty/Views/Custom/CollectionViewCells/BrickCells/Look/SetBackgroundAndWaitBrickCell.swift]: Done.

**Review by zwerg4 — APPROVED** [review]

**Comment by m-herold** [comment]
Thanks @Bomml!

**Pull request merged**